gatorgrouper/apps.py

```diff
@@ -0,0 +1,8 @@
+""" This is undocumented """
+from django.apps import AppConfig
+
+
+class GatorgrouperConfig(AppConfig):
+    """ This is undocumented """
+
+    name = "gatorgrouper"
```

Comment:
@barrezuetai  @sutterj Will changing `name=gatorgrouper` here allow us to rename the app folder to something else?